packages/utilities/psammead-test-helpers/index.test.jsx

```diff
@@ -232,4 +233,14 @@ describe('Psammead test helpers', () => {
     'should match the snapshot for the test component with helmet and other content',
     <HelmetWithContent />,
   );
+
+  it('should create a snapshot from an async it() block with component and done props', async done => {
```

Comment:
you don't need to call done when using async/await https://jestjs.io/docs/en/tutorial-async#async-await

Reply:
I didn't think it was possible to use async/await in Psammead https://github.com/bbc/psammead/issues/2126

I haven't tried this release yet but I suspect this package is broken if the abpve issue hasn't been sorted

Reply:
As the async/await is only in the tests it shouldn't be affected by #2126